### PR TITLE
[Doc] Move contributing part into own file

### PR DESCRIPTION
**Pull request by eiabea**
Comply with the overall open source best practices

### CODE REVIEW

**Comment by lemonmojo** [comment]
@eiabea Shouldn't we at least link to the new contributions doc from the readme?

File: README.md [hunk 1]
```diff
@@ -17,21 +17,3 @@ In case...
 - ...or having great ideas for new content
 
 ... please head over to the issues section and [report an issue](https://github.com/royalapplications/toolbox/issues). Support is provided on a best-effort basis from us and the original script authors.
-
```
Comment (lemonmojo) [inline README.md:20]: @eiabea Please add a link to CONTRIBUTING.md in the readme.

Comment (eiabea) [reply README.md:20]: Added in e97292ff2b92d1b114e427cc8a8cbccfb5af314e

**Review by lemonmojo — APPROVED** [review]
LGTM

**Pull request merged**